rt/bindings/corba/src/main/java/org/apache/cxf/binding/corba/utils/CorbaFixedAnyImplGenerator.java

```diff
@@ -0,0 +1,277 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.cxf.binding.corba.utils;
+
+import org.apache.cxf.Bus;
+import org.apache.cxf.common.spi.ClassGeneratorClassLoader;
+import org.apache.cxf.common.util.ASMHelper;
+import org.apache.cxf.common.util.ASMHelperImpl;
+import org.apache.cxf.common.util.OpcodesProxy;
+
+public class CorbaFixedAnyImplGenerator extends ClassGeneratorClassLoader {
+
+    public CorbaFixedAnyImplGenerator(Bus bus) {
+        super(bus);
+    }
+    public Class<?> createFixedAnyClass() {
+        //TODO move to bus.getExtension(ASMHelper.class)
```

Comment:
Seems like you could use `bus.getExtension(ASMHelper.class)`, correct?

Reply:
Yes but no bus available.... So need refractor to get it...

Reply:
it is in the constructor so it is available ;)